src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The class `Square` should not begin with an empty line after the class declaration. Please remove the empty line to adhere to the coding standards.